packages/expo-image/android/src/main/java/expo/modules/image/decodedsource/DecodedModel.kt

```diff
@@ -0,0 +1,5 @@
+package expo.modules.image.decodedsource
+
+import android.graphics.drawable.Drawable
+
+class DecodedModel(val drawable: Drawable)
```

Comment:
why do you need this empty class?

Reply:
It is used as an input for a Glide to trigger the correct decoder. Glide binds models with decoders to produce resources.